docs/pages/branding/mui-x.tsx

```diff
@@ -0,0 +1,923 @@
+          Spent the morning going through the docs for Material-UI. Such an amazing framework with
+          amazing documentation! I tried using it several months ago, but struggled to grasp how a
+          lot of it worked. Apparently, my understanding of React has been upgraded since then.
+          Smiling face with smiling eyes.
+        </React.Fragment>
+      ),
+    },
+  ],
+];
+
+function WhatCommunitySay() {
+  return (
+    <Container sx={{ px: { xs: 2, sm: 7.5, lg: 3 }, pt: { lg: 15 }, pb: { lg: 20 } }}>
+      <Typography
+        variant="h2"
+        align="center"
+        sx={{ mt: { xs: 12, sm: 15, lg: 0 }, mb: { xs: 4, sm: 10, lg: 8 } }}
+      >
+        What our community
+        <Box component="span" sx={{ display: { xs: 'none', sm: 'block' } }} /> has to say
+      </Typography>
+      <Box sx={{ display: { xs: 'none', lg: 'block' } }}>
+        <Grid container spacing={3}>
+          {communityData.map((community, i) => (
+            <Grid item lg={4} key={i}>
+              {community.map((data, j) => (
+                <CommunitySayCard
+                  key={j}
+                  uniqueKey={j}
+                  name={data.name}
+                  id={data.id}
+                  description={data.description}
+                  avatar={data.avatar}
+                  url={data.url}
+                />
+              ))}
+            </Grid>
+          ))}
+        </Grid>
+      </Box>
+      <Box sx={{ display: { xs: 'block', lg: 'none' } }}>
+        <div className="slider">
+          {communityData.flat().map((data, j) => (
+            <section key={j}>
+              <CommunitySayCard
+                key={j}
+                uniqueKey={j}
+                name={data.name}
+                id={data.id}
+                description={data.description}
+                avatar={data.avatar}
+                url={data.url}
+              />
+            </section>
+          ))}
+          <style>
+            {`
+              .slider {
+                font-family: sans-serif;
+                scroll-snap-type: x mandatory;
+                display: flex;
+                -webkit-overflow-scrolling: touch;
+                overflow-x: scroll;
+                margin: 0 -15px;
+              }
+              section {
+                min-width: 50%;
+              scroll-snap-align: start;
+              position: relative;
+              padding: 0 15px;
+              }
+              @media(max-width:767px){
+                section{
+                      min-width: 100%;
+                }
+              }
+              `}
+          </style>
+        </div>
+      </Box>
+    </Container>
+  );
+}
+// End WhatCommunitySay section
+
+// Start RoadMap section
+const roadMapData = [
+  {
+    image: '/static/branding/mui-x/in-lab.svg',
+    color: 'primary.main',
+    label: 'In the lab',
+    description: <React.Fragment>In the lab, in progress to move into the core</React.Fragment>,
```

Comment:
```suggestion
    description: 'In the lab, in progress to move into the core',
```

Reply:
Done. I removed it where I could. Some places use `<br />` so I needed to keep it. I tried removing the `<br />` but it doesn't look so good.